include/revng/Yield/Internal/BasicBlock.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
```

Comment:
Can you find a more appropriate name than `Internal`? This seems like a relic from caliban.

Reply:
To me it makes sense to have the file that defines `yield::BasicBlock` in `include/revng/Yield/BasicBlock.h`.
Those are data structures as valid as EFA's ones.
All our data structures are always WIP, I wouldn't worry too much.

Reply:
Yeah, thinking about it, I'm not opposed to dropping the `Internal` "namespace".

Reply:
[Done](https://github.com/Cvelth/revng/tree/snapshot/wip_cfg_in_svg/include/revng/Yield).